projectile.el

```diff
@@ -822,7 +832,7 @@ A thin wrapper around `file-truename' that handles nil."
 (defun projectile-get-project-directories ()
   "Get the list of project directories that are of interest to the user."
   (-map (lambda (subdir) (concat (projectile-project-root) subdir))
-        (or (car (projectile-parse-dirconfig-file)) '(""))))
+        (or (nth 0 (projectile-parse-dirconfig-file)) '(""))))
```

Comment:
This is to make it consistent with other parts of the code. We have `(nth 0 x)`, `(nth 1 x)` and `(nth 2 x)`. It seems cleaner to use the same kind of extractors everywhere.